src/seedu/addressbook/Person.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * 
+ */
+package seedu.addressbook;
+
+/**
+ * @author YiMin
+ *
+ */
+public class Person {
+    private static final String PERSON_STRING_REPRESENTATION = "%1$s " // name
+            + AddressBook.PERSON_DATA_PREFIX_PHONE + "%2$s " // phone
+            + AddressBook.PERSON_DATA_PREFIX_EMAIL + "%3$s"; // email
+	
+	private final String name_;
+	private final String email_;
+	private final String phone_;
+	
+	public Person(String name, String phone, String email){
+		name_=name;
+		phone_=phone;
+		email_=email;
+	}
+
+	/**
```

Comment:
For getter and setter, it is okay to omit the comments as they are considered quite clear.